lib/manageiq/providers/nuage/engine.rb

```diff
@@ -11,6 +11,12 @@ class Engine < ::Rails::Engine
           app.config.paths["config/secrets"] << root.join("config", "secrets.yml").to_s
         end
 
+        initializer 'manageiq-providers-nuage.configure_autoloader' do
+          # The following file should not be eager loaded as it depends on being installed, which isn't possible on macs and isn't done by default
+          # in ci for other plugins, core, etc.
+          Rails.autoloaders.main.ignore(root.join("app/models/manageiq/providers/nuage/network_manager/event_catcher/messaging_handler.rb")) unless Gem.loaded_specs.key?(:qpid_proton)
```

Comment:
Interesting seems like this should be a string key `Gem.loaded_specs.key?("qpid_proton")`  Wonder when that changed, we should change openstack if it is an issue there as well.
With that it is green for me locally.